src/components/Player/GridControls.js

```diff
@@ -154,12 +155,65 @@ export default class GridControls extends Component {
     }
   };
 
+  _handleKeyDownVim = (key, shiftKey) => {
+    const actionKeys = {
+      ArrowLeft: 'left',
+      ArrowUp: 'up',
+      ArrowDown: 'down',
+      ArrowRight: 'right',
+      Backspace: 'backspace',
+      '{del}': 'backspace',
+      Delete: 'delete',
+      Tab: 'tab',
+      ' ': 'space',
+      '[': 'backward',
+      ']': 'forward',
+    };
+
+    const normalModeActionKeys = {
+      h: 'left',
+      j: 'down',
+      k: 'up',
+      l: 'right',
+      x: 'delete',
+      ' ': 'space',
+      '[': 'backward',
+      ']': 'forward',
+    };
+
+    const {onVimNormal, onVimInsert, vimInsert, onPressEnter, onPressPeriod} = this.props;
+    if (key in actionKeys) {
+      this.handleAction(actionKeys[key], shiftKey);
```

Comment:
return true to prevent tab default behavior

Reply:
i noticed this bug but was having trouble figuring out why it was happening. thanks!